README.md

```diff
@@ -117,20 +122,6 @@ To train the 2.8B param model:
 
 - Instead, only set `per-device-train-batch-size` and `per-device-eval-batch-size` to 3 in the `train_dolly.py` invocation of `deepspeed`
 
-#### V100 GPUs
```

Comment:
Let's leave the section headers for visibility. It can however just direct you to select 'v100' and summarize what is different.

Reply:
Done